src/debug/FabricDebugConfigurationProvider.ts

```diff
@@ -30,6 +30,43 @@ export abstract class FabricDebugConfigurationProvider implements vscode.DebugCo
 
     private runtime: FabricRuntime;
 
+    public static async getInstantiatedVersion(chaincodeName: string): Promise<{ name: string, version: string }> {
```

Comment:
Could we change the function name to `getInstantiatedChaincode` or something instead, as it returns the name as well as the version.